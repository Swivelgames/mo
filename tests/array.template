{{#repo}}
  <b>{{@index}} - {{.}}</b>
{{/repo}}
