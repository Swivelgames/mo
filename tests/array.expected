  <b>0 - resque</b>
  <b>1 - hub</b>
  <b>2 - rip</b>
